tests/test_fetch.py

```diff
@@ -287,3 +288,41 @@ def test_query_caching(self):
 
         # reset cache directory state (will fail if purge was unsuccessful)
         os.rmdir(os.path.expanduser('~/dj_query_cache'))
+
+    def test_fetch_group_by(self):
+        # https://github.com/datajoint/datajoint-python/issues/914
+
+        expected_data = [{'parent_id': 1}]
+        fetched_data = Parent().fetch('KEY', order_by='name')
+        print(fetched_data)
+        assert fetched_data == expected_data
+
+    def test_dj_U_DISTINCT(self):
```

Comment:
Following python's `snake_case` convention for functional naming.

```suggestion
    def test_dj_u_distinct(self):
```

Otherwise, a great test. :+1: